pkg/token/token.go

```diff
@@ -523,6 +525,7 @@ func (v tokenVerifier) Verify(token string) (*Identity, error) {
 		return nil, NewSTSError(fmt.Sprintf("error reading HTTP result: %v", err))
 	}
 
+	metrics.Get().StsResponseErrorCode.WithLabelValues(fmt.Sprint(response.StatusCode)).Inc()
```

Comment:
but do we care about 200s?  is 200 an error code? why is the metric name sts_responses_total if the variable name is sts_response_error_code (it's hard to tell intention from the naming)

Reply:
Yeah we should probably either rename `StsResponseErrorCode` or only increment for `response.StatusCode != 200`.  The benefit of including successes is the ability of getting success % metric.

Reply:
So the first option would be to just rename `StsResponseErrorCode` to `StsResponseCode`.

Reply:
yes..i missed that

Reply:
done..changed the variable name